accountREADME.md

```diff
@@ -0,0 +1,181 @@
+# My Account: Phase 1
+
+This README details the internal Next.js API endpoints that hit Sierra API, which now support [Research Catalog's My Account page](https://www.nypl.org/research/research-catalog/account) (instead of the webscraping previously used). See [Sierra API documentation](https://sandbox.iii.com/iii/sierra-api/swagger/index.html#!/patrons) for more information.
+
+## Account API Endpoints
+
+In general, successful requests will return a `200` status. Requests will fail with a `403` status if the patron is not authenticated, or if the patron does not own the item they are updating. These failure responses will look like this:
+
+```
+exampleFailureResponse: {
+    status: 403,
+    message: 'No authenticated patron'
+}
+exampleFailureResponse: {
+    status: 403,
+    message: 'Authenticated patron does not own this hold'
+}
+
+```
+
+If the request fails because it's rejected by Sierra (a common example is a checkout being renewed too many times), it will return a `4XX` status, with a message from Sierra.
```

Comment:
You also get the status from the rejected Sierra request? It's mostly likely a 400 bad request or something else? Not for this PR but either return whatever Sierra returns or set a status other than 403 for bad Sierra requests.